### ISSUE TITLE
role toggling

### ISSUE THREAD

**Issue by rainbreak** [dapphub/ds-roles#15]
Consider the ability to toggle a role's authentication across all of its functions.

**Comment by rainbreak** [comment]
Then could have e.g. `0` as the special 'public' role, instead of `public_capabilities`

**Comment by gbalabasquer** [comment]
I'm still not getting the advantage of it. We would still need to add to this special role each function we would want to be accessible by it. Wouldn't be the same?

**Comment by rainbreak** [comment]
Yes, you still have to do that initially. But then toggling the role's access is a single call operation.

**Comment by gbalabasquer** [comment]
We could still have a flag to enable/disable the `public_capabilities`, but it is true this would be more generic and would allow to toggle any role.
